draft-ietf-rtcweb-jsep.xml

```diff
@@ -1520,15 +1520,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             </list></t>
 
           <t>The next step is to generate m= sections, as specified in
-          <xref target="RFC4566"/> Section 5.14, for each
-          RtpTransceiver that has been added to the PeerConnection via
-          the addTrack or addTransceiver methods, excluding stopped
-          RtpTransceivers that are not associated with an m= section
-          (either due to an m= section being recycled or an
-          RtpTransceiver having been stopped before being associated
-          with an m= section), m= sections are generated in the order
+          <xref target="RFC4566"/> Section 5.14.  An m= section is
+          generated for each RtpTransceiver that has been added to the
+          PeerConnection via the addTrack, addTransceiver, and
+          setRemoteDescription methods.  This is done in the order
```

Comment:
I realized that since the 'subsequent offer' text says follow the same steps here, we need to reference setRemoteDescription here, which can also create a transceiver, even if it's a little awkward to mention that for the initial offer.